pkgs/flutter_http_example/lib/http_client_factory.dart

```diff
@@ -7,13 +7,23 @@ import 'dart:io';
 import 'package:cronet_http/cronet_http.dart';
 import 'package:cupertino_http/cupertino_http.dart';
 import 'package:http/http.dart';
+import 'package:http/io_client.dart';
+
+const maxCacheSize = 2 * 1024 * 1024;
```

Comment:
[nit] This is in a `lib/` library in the example - was the intent to demonstrate including `maxCacheSize` in the example package public API?

Consider making this private.

Reply:
No, this should be private.